libshaderc_util/include/libshaderc_util/compiler.h

```diff
@@ -31,10 +32,68 @@
 namespace shaderc_util {
 
 // Initializes glslang on creation, and destroys it on completion.
+// There is expected to be precisely one of these objects active within
```

Comment:
Actually, there can be zero. :)

Also, "active" could be ambiguous.  Perhaps it's better to say that this is thread-hostile and must be a singleton.